### ISSUE TITLE
input and logits tesnor names

### ISSUE THREAD

**Issue by asgsaeid** [tensorflow/models#5471]
how to get input and logits (one layer before last activation) tensors names of the following pre-trained model?

http://download.tensorflow.org/models/deeplab_cityscapes_xception71_trainfine_2018_09_08.tar.gz

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
